### ISSUE TITLE
segfault on `\RdKafka\Metadata\Topic::getTopic()`

### ISSUE THREAD

**Issue by ctrl-f5** [php-rdkafka/php-rdkafka#465]
We have the following code running during deploy to verify required kafka topics exist:

```php
$consumer = new \RdKafka\KafkaConsumer(); // this is built up elsewhere
$existingTopics = $consumer
    ->getMetadata(true, null, 1000)
    ->getTopics();

foreach ($existingTopics as $topic) {
    $this->availableTopicsInKafka[] = $topic->getTopic();
}
```

Which works perfectly on 7.4.x but fails consistently on 8.0-alpine with a segementation fault on the first call to `$topic->getTopic()`.

> Segmentation fault (core dumped)

I tried enabling librdkafka logs and debug but it doesnt output anything.

```
PHP Version => 8.0.9

System => Linux 81bdef77c496 5.11.0-27-generic #29~20.04.1-Ubuntu SMP Wed Aug 11 15:58:17 UTC 2021 x86_64

rdkafka support => enabled
version => 5.0.0
build date => Sep  3 2021 09:45:22
librdkafka version (runtime) => 1.7.0
librdkafka version (build) => 1.7.0.255
```

**Comment by rafelangelo** [comment]
We face the same problem while migrating vom 7.4 to 8.0 on Alpine 3.13. It is possible to do a count() on the RdKafka\Metadata\Collection, but accessing the items is not possible.


```
PHP 8.0.10 (cli) (built: Sep  1 2021 04:23:01) ( NTS )

rdkafka

rdkafka support => enabled
version => 5.0.0
build date => Sep  2 2021 22:08:41
librdkafka version (runtime) => 1.7.0
librdkafka version (build) => 1.7.0.255
```

**Comment by dirx** [comment]
Hi @ctrl-f5, after some testing - looks like keeping `$metadata` seems to work for us as workaround:

```php
$consumer = new \RdKafka\KafkaConsumer(); // this is built up elsewhere
$metadata = $consumer->getMetadata(true, null, 1000);

foreach ($metadata->getTopics() as $topic) {
    $this->availableTopicsInKafka[] = $topic->getTopic();
}
```

**Comment by arnaud-lb** [comment]
Thanks @ctrl-f5 @rafelangelo @dirx 

This should be fixed by #477
